lmn/views/views_users.py

```diff
@@ -21,22 +21,31 @@ def my_user_profile(request):
     # TODO - editable version for logged-in user to edit their own profile
```

Comment:
remove this todo comment if this is done

Reply:
Ok. I will do that